File: nature-harvest-frontend/src/app/dtos/payment/momo-payment.dto.ts
import { OrderDto } from '../order/order.dto';

export class MoMoPaymentDto {
  requestType: string;
  lang: string;
  order: OrderDto;

  constructor(data: any) {
    this.requestType = data.requestType;
    this.lang = data.lang;
    this.order = data.order;
  }
}

File: nature-harvest-backend/nature-harvest-backend/src/main/java/com/api/nature_harvest_backend/dtos/payment/MomoPaymentDto.java
package com.api.nature_harvest_backend.dtos.payment;

import com.api.nature_harvest_backend.dtos.order.OrderDto;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class MomoPaymentDto {
    private String requestType;
    private String lang;
    private OrderDto order;
}
